Content.Shared/Construction/Prototypes/ConstructionPrototype.cs

```diff
@@ -14,89 +12,80 @@ public sealed partial class ConstructionPrototype : IPrototype
     /// <summary>
     ///     Hide from the construction list
     /// </summary>
-    [DataField("hide")]
+    [DataField]
     public bool Hide = false;
 
     /// <summary>
     ///     Friendly name displayed in the construction GUI.
     /// </summary>
     [DataField("name")]
-    public string Name = string.Empty;
+    public LocId? NameLocId;
+
+    public string? Name;
 
     /// <summary>
     ///     "Useful" description displayed in the construction GUI.
     /// </summary>
     [DataField("description")]
-    public string Description = string.Empty;
+    public LocId? DescLocId;
+
+    public string? Description;
```

Comment:
```suggestion
    public LocId? Description;
```

Use only the LocID field so that your work doesn't go down the drain when someone decides to add non-localized text.